src/dotnet/dotnet.csproj

```diff
@@ -1,6 +1,5 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
MSBuild implemented a change to preserve whitespace https://github.com/Microsoft/msbuild/pull/1036 and CPS needs to consume it to stop VS from messing up the file.